docs/check-ojp-schemas.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
This script must be added to https://github.com/VDVde/OJP/blob/changes_for_v1.1/.travis.yml#L7, `xsltproc` is already defined as a dependency to be installed here https://github.com/VDVde/OJP/blob/changes_for_v1.1/.travis.yml#L4